README.md

```diff
@@ -23,7 +23,51 @@ juju add-unit -n 2 etcd
 
 Its recommended to run an odd number of machines as it has greater redundancy than even number (ie. 4, you can lose 1 before quorum is lost, where as 5, you can 2).
 
+# The charm told me to see the README
 
+## Usage with your own binaries
+
+This charm supports resources, which means you can supply your own release of
```

Comment:
'can'.  Its option as you may be running on PPC64EL architecture and need to supply a non AMD64 bin for the workload to function